real_robots/omnirobot_utils/marker_finder.py

```diff
@@ -0,0 +1,386 @@
+import numpy as np
+import cv2
+from os.path import join
+import yaml
+import argparse
+
+
+def rotateMatrix90(matrix):
+    """
+
+    :param matrix: (array) matrix to rotate of 90 degrees
+    :return: (array)
+    """
+    new_matrix = np.transpose(matrix)
+    new_matrix = np.flip(new_matrix, axis=1)
+    return new_matrix
+
+
+def hammingDistance(s1, s2):
+    """
+
+    :param s1: (str)
+    :param s2: (str)
+    :return: (int) Hamming distance between s1 & s2
+    """
+    assert len(s1) == len(s2)
+    return sum(ch1 != ch2 for ch1, ch2 in zip(s1, s2))
+
+
+class MakerFinder():
```

Comment:
@GaspardQin I am not sure what the functions in this file do (use case of script also) ,  please can you comment ? (see TODO)

Reply:
Actually, we can delete makerfinder class, it is not used in this version (but perhaps it will help in the future if the marker is changed to some other things)

Reply:
Alright, then please help comment the class (see TODO markers in the file).
We can keep it as we might modify the env/task for future exps.